packages/base/zarf.yaml

```diff
@@ -10,6 +10,11 @@ metadata:
   version: "0.40.0"
   # x-release-please-end
   x-uds-dependencies: []
+  annotations:
+    dev.uds.title: UDS Core (Base)
+    dev.uds.tagline: Provides the base for all other UDS Core Functional Layers
+    dev.uds.category: UDS-Core
+    dev.uds.keywords: uds,pepr,istio
```

Comment:
operator and policies might be good keywords here too but not a big deal either way.